helix-term/src/commands.rs

```diff
@@ -4839,7 +4844,7 @@ fn find_next_char_until_newline<M: CharMatcher>(
 }
 
 /// Decrement object under cursor by `amount`.
-fn increment_impl(cx: &mut Context, amount: i64) {
+fn increment_impl(cx: &mut Context, amount: i64, increment_direction: IncrementDirection) {
```

Comment:
You can use the direction parameter and `cx.count()` to determine the amount in this function and then `amount` doesn't need to be a parameter.

For example, `increment` can be implemented just as

```rust
fn increment(cx: &mut Context) {
    increment_impl(cx, IncrementDirection::Increase);
}
```